easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.3.2-foss-2016b.eb

```diff
@@ -0,0 +1,26 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
```

Comment:
I don't really know if each EasyConfig should be cluttered with a long list of contributors. I think this is what git is for. But you do you I guess.

Is there any prefered method on how to add attributions to an EasyConfig? I found several formats in different files, I would stick to the one found in the config. Maybe there should be a guideline/example on how to format/structure EasyConfigs?